telemetry/jetbrains/src/test/resources/testOverrideOutput

```diff
@@ -68,6 +78,12 @@ public object MetadataTelemetry {
                 unit(Unit.NONE)
                 value(value)
                 passive(passive)
+                if(reason != null) {
+                    metadata("reason", reason)
+                }
+                if(result != null) {
```

Comment:
they cannot be grouped in the same if statement?

Reply:
this is unit test output